xbmc/utils/StreamDetails.cpp

```diff
@@ -42,6 +43,13 @@ CStreamDetailVideo::CStreamDetailVideo() :
 {
 }
 
+CStreamDetailVideo::CStreamDetailVideo(const VideoStreamInfo &info, int duration) :
```

Comment:
Can you push each initialization on a new line? This scales better with cleaner history when adding additional fields.